velox/type/Tokenizer.h

```diff
@@ -51,6 +53,7 @@ class Tokenizer {
   const char UNICODE_CARET = '^';
 
   const std::string path_;
+  std::vector<char> regularChars_;
```

Comment:
Instead of customizing what is regular, I think we should customize field name separator instead.  What does spark use to separate field names?